### ISSUE TITLE
mutation mapper selection button behavior on first click

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#6883]
Would it be possible to have the first click select the mutation type you are clicking on and deselect everything else? Currently the mutation type you click on becomes unselected with everything else selected, which is the opposite of what the behavior was before

**Comment by onursumer** [comment]
Yes, we can do something like this:

```
if (all selected)
    deselect all except the clicked one
else
    toggle the clicked one
```

**Comment by onursumer** [comment]
fixed by cBioPortal/cbioportal-frontend/pull/2923
